### DISCUSSION TITLE
Generate new boundary (sideset) from current boundary

### DISCUSSION THREAD

**Discussion by lyyc199586** [idaholab/moose#24350] [Q&A Meshing] [answered]
Hi all,

I am trying to find a way to generate a new boundary from an existing boundary. To be specific, I want to 'cut' part of the exist boundary:

![null](https://github.com/idaholab/moose/assets/20653957/31e793b7-47de-49d6-bd31-88121d5a47c3)

Is there an easy way to do so?

**Comment by GiudGiud** [comment]
Hello

If you know a good analytical expression for it, like "x > 3.2" for example, then you can use the ParsedGenerateSideset
https://mooseframework.inl.gov/source/meshgenerators/ParsedGenerateSideset.html

Guillaume

**Reply by lyyc199586** [reply]
Can I use old boundary with the expression "x > 3.2" to generate a new boundary with ParsedGenerateSideset?

**Reply by GiudGiud** [reply]
unfortunately not.
It's definitely something we're missing

**Reply by lyyc199586** [reply]
I feel like it shouldn't be too hard to add an option `included_boundaries` similar to `included_subdomians`. Let me try to add it

**Comment by GiudGiud** [comment, marked as answer]
Functionality was added
